### PR TITLE
Fix that dict-ids in examples caused server failure

### PR DESCRIPTION
**Pull request by almarklein**
In #1039 I encountered an odd bug: if I navigated to the new chapter it worked fine, but if I refreshed the page, or navigate to the page by entering the url, then I got a descriptive error message, see the image below.

I added some code to the point of failure to see what the value of `attrib` was at that point, and found it was a dictionary-id of a component in one of the examples (well, in all examples, really). 

Fortunately, the fix is easy, I can confirm that the docs in #1039 don't have the bug with the change proposed here. 

afbeelding

### CODE REVIEW

**Comment by almarklein** [comment]
Quick note, in case people wonder: `bool` is a subclass of `int`.

File: dash_docs/convert_to_html.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _translate_attrib(attrib):
         False: 'false',
         'False': 'false',
     }
-    if attrib in attrib_map:
+    if isinstance(attrib, (str, int)) and attrib in attrib_map:
```
Comment (chriddyp) [inline dash_docs/convert_to_html.py:63]: I'm not sure I follow - What about when the attrib is `True` or `False` as included above?

Comment (chriddyp) [reply dash_docs/convert_to_html.py:63]: Ah I see your comment above - `bool` is a subclass of `int`. TIL!

Comment (chriddyp) [reply dash_docs/convert_to_html.py:63]: Could you just add a comment above this line saying that `bool is a subclass of int`?

**Review by chriddyp — APPROVED** [review]
💃🏼 once my comment is considered

File: dash_docs/convert_to_html.py (hunk 1 above)
Comment (almarklein) [reply dash_docs/convert_to_html.py:63]: That works, because `bool` is a subclass of `int` in Python.

Comment (almarklein) [reply dash_docs/convert_to_html.py:63]: I'll just add `bool` to the list to avoid confusion ...

Comment (almarklein) [reply dash_docs/convert_to_html.py]: GH comments are not fast enough to function as a chat, haha. I can undo that last change and add a comment, if you prefer.

Comment (almarklein) [reply dash_docs/convert_to_html.py:64]: Updated to use a comment instead.

**Pull request merged**